Weather App/viewModel/WeatherForCityViewModel.swift

```diff
@@ -4,25 +4,24 @@ import Keys
 
 protocol WeatherForCityViewModel {
     var weathers: [WeatherForCity] { get }
+    var weatherForLocation: WeatherForCity? { get }
     func getWeatherForCity(cityName: String) async
     func getWeatherForCoordinate(coordinate: CLLocationCoordinate2D) async
 }
 
 class WeatherForCityViewModelImpl: WeatherForCityViewModel {
     var weathers: [WeatherForCity] = []
+    var weatherForLocation: WeatherForCity?
 
     func getWeatherForCity(cityName: String) async {
-        let city = cityName.removeDiacratics()
-        let isContained = weathers.contains {
-            $0.name.removeDiacratics() == city
-        }
-        guard !isContained, let requestURL = Bundle.main.object(forInfoDictionaryKey: "WeatherRequestURL") as? String,
-              let url = URL(string: requestURL + "?q=\(city)" +
+        let cityName = cityName.removeDiacratics()
```

Comment:
Maybe there will be better to store cities names in storage without diacritics. We can also store two values inside a model, one for displaying and second for API. Btw, here is typo, should be `removeDiacritics `

Reply:
👍